File: Iniciante/C/1018.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main()
{

    int notas, resto, valor;

    scanf("%d", &valor);

    printf("%d\n", valor);

    notas = valor / 100;
        printf("%d nota(s) de R$ 100,00\n", notas);

    resto = valor % 100;

    notas = resto / 50;
        printf("%d nota(s) de R$ 50,00\n", notas);

    resto = resto % 50;

    notas = resto / 20;
        printf("%d nota(s) de R$ 20,00\n", notas);

    resto = resto % 20;

    notas = resto / 10;
        printf("%d nota(s) de R$ 10,00\n", notas);

    resto = resto %10;

    notas = resto / 5;
        printf("%d nota(s) de R$ 5,00\n", notas);

    resto = resto %5;

    notas = resto /2;
            printf("%d nota(s) de R$ 2,00\n", notas);

    resto = resto % 2;

    notas = resto / 1;
        printf("%d nota(s) de R$ 1,00\n", notas);




    return 0;
}

File: Iniciante/Python/1018.py
valor = int(input()) #576

print(valor)

notas = valor // 100 #5 notas
resto = valor % 100 #sobra 76
valor = resto
print(notas, "nota(s) de R$ 100,00")

notas = valor // 50 #5
resto = valor % 50 #76
valor = resto
print(notas, "nota(s) de R$ 50,00")

notas = valor // 20 #5
resto = valor % 20 #76
valor = resto
print(notas, "nota(s) de R$ 20,00")

notas = valor // 10 #5
resto = valor % 10 #76
valor = resto
print(notas, "nota(s) de R$ 10,00")

notas = valor // 5 #5
resto = valor % 5 #76
valor = resto
print(notas, "nota(s) de R$ 5,00")

notas = valor // 2 #5
resto = valor % 2 #76
valor = resto
print(notas, "nota(s) de R$ 2,00")

notas = valor // 1 #5
resto = valor % 1 #76
print(notas, "nota(s) de R$ 1,00")
